Header header
uint32 ID
uint64 timestamp # Wann das Quellbild gemacht wurde
float64 xPosition
float64 yPosition
uint32 quadcopterId
